src/main/kotlin/uk/gov/justice/digital/hmpps/authorizationserver/data/model/Client.kt

```diff
@@ -63,19 +61,16 @@ data class Client(
   var mfaRememberMe: Boolean,
 
   var mfa: MfaAccess?,
+
+  var databaseUsername: String?,
+
+  var jiraNum: String?,
```

Comment:
Sorry, just a minor thing, but can we re-name this to just `jira`, instead of `jiraNum` - also rename the database field the same